test/server.jl

```diff
@@ -251,24 +251,110 @@ end
     @test HTTP.Servers.shutdown(nothing) === nothing
 
     IOserver = Sockets.listen(Sockets.InetAddr(parse(IPAddr, "127.0.0.1"), 8052))
-    
+
     # Shutdown adds 1
     TEST_COUNT = Ref(0)
     shutdown_add() = TEST_COUNT[] += 1
-    server = HTTP.Servers.Server(nothing, IOserver, "host", "port", shutdown_add)
+    server = HTTP.Servers.Server(nothing, IOserver, "host", "port", shutdown_add, nothing)
```

Comment:
Why does this not default to `nothing`?
Isn't making this argument required going to be a breaking change?

Reply:
I mean, this is pretty internal I would think? The next to last argument was added pretty recently too. Doesn't hurt to be conservative though